crates/eww/src/config/inbuilt.rs

```diff
@@ -64,6 +64,12 @@ define_builtin_vars! {
 
     // @desc EWW_TIME - the current UNIX timestamp
     "EWW_TIME" [1] => || Ok(DynVal::from(get_time())) ,
+
+    // @desc EWW_IPV4 - Information about the IPv4 address on all interfaces except "lo"
+    "EWW_IPV4" [1] => || Ok(DynVal::from(get_ipv4())),
```

Comment:
duration should be 2 seconds as with every other variable except time
i'm not sure whether running this that often is even needed